R/timeseriessummary-data.R

```diff
@@ -323,6 +323,7 @@ parseTSSRatingCurves <- function(reportData, timezone){
     
     curves <- curves[-which(names(curves) == "shifts")]
     curves <- curves[-which(names(curves) == "baseRatingTable")]
+    curves <- curves[-which(names(curves) == "offsets")]
```

Comment:
the offsets column wasn't part of the output anyway, so I believe it won't have any unintended consequences for rating curve output